docs/en/sql-reference/statements/select/into-outfile.md

```diff
@@ -4,10 +4,35 @@ toc_title: INTO OUTFILE
 
 # INTO OUTFILE Clause {#into-outfile-clause}
 
-Add the `INTO OUTFILE filename` clause (where filename is a string literal) to `SELECT query` to redirect its output to the specified file on the client-side.
+`INTO OUTFILE` clause redirects the result of a `SELECT` query to a file on the **client** side.
+
+Compressed files are supported. Compression type is detected by the extension of the file name. Or it can be explicitly specified in a `COMPRESSION` clause.
```

Comment:
```suggestion
Compressed files are supported. Compression type is detected by the extension of the file name (mode `'auto'` is used by default). Or it can be explicitly specified in a `COMPRESSION` clause.
```